capstone/src/main/java/com/google/sps/Objects/response/PollResponse.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.Objects.response;
+
+import java.util.List;
+
+import com.google.sps.Objects.Option;
+
+public final class PollResponse {
+
+  private final List<Option> options;
+  // List of ids of options for which current user voted
+  private final List<Long> votedOptions;
```

Comment:
I discussed this with Andrew, but what I'm trying to do is create a PollResponse which is a wrapper class that contains information that I need that will be converted to json. I need to know which options the current logged in user has voted for so that when I load the page, I know which options to mark as checked. I want to get all the information I need with one fetch request from PollServlet, so even if I attributed this with User, I would still need another wrapper class to put together the poll and user information. In this case, I don't think I need to put this info with the User object because the User object also contains a lot of information that I will not be needing.

Reply:
Oh okay I see now, I didn't understand that this was the response, thank you for the clarification!